docs/guide/acquiring_data/database.rst

```diff
@@ -325,6 +325,19 @@ resulting entries to the database. Because the query result translates to
     >>> len(database)
     37
 
+There is another level of caching which ensures a previously downloaded file
```

Comment:
No old caching is still there
`https://github.com/sunpy/sunpy/pull/1785/files#diff-89ea9c1fe40fcd53bc37b328a8ec5e6fL449`